Arduino/libraries/sensors/sensors.h

```diff
@@ -0,0 +1,31 @@
+#ifndef sensors_h
+#define sensors_h
+#include "Arduino.h"
+#include "HCSR04.h"
+#include "MPU6050.h"
+#include "I2Cdev.h"
+#include "Adafruit_ICM20X.h"
+#include "Adafruit_ICM20948.h"
+#include "Adafruit_Sensor.h"
+class Sensors {
+  public:
+    Sensors(int trig_pin, int echo_pin[3], int num_sensors);
+    void readUltrasonic(HCSR04 *hc, float usReadings[3]);
+    int16_t * readIMU(MPU6050 accelgyro);
+    bool calibrateUltrasonic(int num_readings, HCSR04 hc);
+    int * calibrateIMU(MPU6050 accelgyro);
```

Comment:
probably wanna delete these when we are sure we are not using it

Reply:
Gonna save this for a later time if we find our current IMU (ICM20948) isn't working. Will keep for now.